src/Ledger/Certs.lagda

```diff
@@ -317,16 +315,17 @@ data _⊢_⇀⦇_,DELEG⦈_ where
   DELEG-delegate : let open PParams pp in
     ∙ (c ∉ dom rwds → d ≡ keyDeposit)
     ∙ (c ∈ dom rwds → d ≡ 0)
+    ∙ mc ∈ mapˢ just (dom dReps) ∪ ❴ nothing ❵
```

Comment:
This is pretty much correct, except that the type needs to be `Maybe VDeleg`. So you also need to map `credVoter DRep` over it.